tests/manual/test-procedure.rst

```diff
@@ -95,17 +99,10 @@ You will need
 Tests Execution
 ---------------
 
-All Test Cases have to be run in one of the browsers listed in the
-"Preparation" section, and AT LEAST the following Test Cases have to be run
-in all of the other browsers as well:
-
-- Hex file can be generated
-- Generated hex file can be loaded to the editor
-- Python file can be saved
-- Saved Python file can be loaded to the editor
-- Snippets inject code into the Text Editor
-- Help menu expands and links work
-- Zoom changes the Text Editor font size
+For a Primary Editor release run all the tests in all browsers. For a Beta
+Editor release, run all tests in Internet Explorer 10, except for the
+"Connect and Flash over WebUSB and use REPL" test case, which has to be run
```

Comment:
The "WebUSB not supported message is working" test case also has to be run on a Chrome-based browser